app/src/main/java/ch/hslu/refashioned/ui/userScore/UserScoreViewModel.java

```diff
@@ -0,0 +1,56 @@
+package ch.hslu.refashioned.ui.userScore;
+
+import android.app.Application;
+import android.graphics.Color;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import java.util.stream.Stream;
+
+import ch.hslu.refashioned.model.history.Brand;
+import ch.hslu.refashioned.model.history.Purchase;
+import ch.hslu.refashioned.repository.history.PurchaseRepo;
+import ch.hslu.refashioned.service.history.MockPurchaseService;
+import ch.hslu.refashioned.service.history.PurchaseService;
+import ch.hslu.refashioned.ui.color.GradientProvider;
+
+public final class UserScoreViewModel extends AndroidViewModel {
+    private final PurchaseService service;
```

Comment:
suggestion: using the score fragment and adding the min max functionality to the interface